src/app/participant-list/participant-list.component.html

```diff
@@ -710,7 +711,7 @@ <h3 class="Color--primary Line--Break">{{additionalMessage}}</h3>
               <b># of participants: {{participantList.length}}</b>
             </td>
           </tr>
-          <tr>
+          <tr *ngIf="assignees != null && assignees.length > 0">
```

Comment:
since this check is used in several places I think it would be good to extract it as a small checker method, which would help to minimize code duplication.